project/Dependencies.scala

```diff
@@ -140,6 +139,7 @@ object Dependencies {
     val sbtOrg = "com.github.sbt"
     val logbackOrg = "ch.qos.logback"
     val awsSdkOrg = "software.amazon.awssdk"
+    val dwickernOrg = "com.github.dwickern"
```

Comment:
Not used.

Reply:
you are right, I incorrectly added this today, post-merge conflict resolution